### PR TITLE
fix(alert): allow 5.5 items to show in alert-mode select popup in md

### PR DESCRIPTION
**Pull request by amandaesmith3**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`npm run build`) was run locally and any changes were pushed
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: Resolves https://github.com/ionic-team/ionic-framework/issues/23760

In `md` mode, alert-style `ion-select`s with more than 5 items don't look scrollable, due to the alert content having the height for exactly 5 items.

## What is the new behavior?

- Alert content height increased to allow for 5.5 items (going off the MD spec item height of 48px). The cut-off item makes it clear that the list can be scrolled to reveal more items.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: core/src/components/alert/alert.md.vars.scss [hunk 1]
```diff
@@ -80,7 +80,7 @@ $alert-md-message-empty-padding-bottom:       $alert-md-message-empty-padding-to
 $alert-md-message-empty-padding-start:        $alert-md-message-empty-padding-end !default;
 
 /// @prop - Maximum height of the alert content
-$alert-md-content-max-height:                 240px !default;
+$alert-md-content-max-height:                 264px !default;
```
Comment (liamdebeasi) [inline core/src/components/alert/alert.md.vars.scss:83]: If we are trying to show 1/2 of the 6th item's height then this should be `266px`. Half the item height is `24px`, but there are 2 `1px` borders to account for which increase the max content height from `264px` to `266px`:

| `264px` | `266px` |
| --------- | -------- |
| Screen Shot 2021-09-24 at 11 56 56 AM | Screen Shot 2021-09-24 at 11 56 15 AM |

```suggestion
$alert-md-content-max-height:                 266px !default;
```

**Review by liamdebeasi — CHANGES_REQUESTED** [review]

**Review by liamdebeasi — APPROVED** [review]
Great job! One final thing: We try to word our commits such that they describe what issue was resolved rather than what changed in the code. So one way you could word this is:

```
fix(alert): made it easier to determine if an alert is scrollable in MD mode
```

However you want to word it is up to you, as long as we convey what issue got fixed.

**Comment by amandaesmith3** [comment]
> Great job! One final thing: We try to word our commits such that they describe what issue was resolved rather than what changed in the code. 

🤔 The contributing doc says to do the reverse: https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#commit-message-guidelines

> describe what the commit does, not what issue it relates to or fixes

**Comment by liamdebeasi** [comment]
Sorry that's what I meant -- My comment was worded in a confusing way. This PR makes it easier to tell if an alert is scrollable, so describing the commit in that manner is something we should shoot for. (At the end of the day it's not an exact science, so if you have a commit message that you think is descriptive for Ionic developers then that's fine).

Thanks for pointing that out!

**Pull request merged**